website_docs/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "C++"
+weight: 11
+description: >
+  A language-specific implementation of OpenTelemetry in C++.
+---
+
+This is the OpenTelemetry for C++ documentation. OpenTelemetry is an
+observability framework -- an API, SDK, and tools that are designed to aid in
+the generation and collection of application telemetry data such as metrics,
+logs, and traces. This documentation is designed to help you understand how to
+get started using OpenTelemetry for C++.
+
+## Status and Releases
+
+The current status of the major functional components for OpenTelemetry C++ is
+as follows:
+
+| Tracing   | Metrics   | Logging             |
+| -------   | -------   | -------             |
+| Pre-Alpha | Pre-Alpha | Not Yet Implemented |
```

Comment:
nit - we do have logging api/sdk pre-alpha release.

Reply:
And I think Metrics should be "Not Yet Implemented".